### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by merspieler** [albar965/littlenavmap#1266] [General]
Thank you for making this wonderful program which I really enjoy using.

**Comment by albar965** [comment]
You're very welcome! :slightly_smiling_face:  :+1:

**Comment by cowreth** [comment]
I'd like to state formally that this particular issue is still open.
I discovered the Beta3, few days ago. A very nice edition, IMHO.

Alex, and contributors, the work you've done is wonderful. It is neat, it helps a lot of pilots. I have the chance to introduce LNM to a lot of persons, and it is easy to make them know why LNM is irreplaceable and useful. From new pilots to very experienced pilot, I can't count the times I've said "yep, you can find this info / make this check thanks to LNM".

Best wishes to you Alex, to the contributors, and to all the friendly users around the world for the end of the year :)

**Comment by albar965** [comment]
@cowreth Seb, this is not really an issue. :slightly_smiling_face: And thank you very much for your kind words! Appreciate it.
Wish you a Happy New Year! :fireworks:

**Comment by gsegit** [comment]
Thanky you for the new Version ;D

Greetings from Germany ;D

Gerriet

**Comment by albar965** [comment]
> Thanky you for the new Version ;D
> 
> Greetings from Germany ;D

You're welcome. :slightly_smiling_face: 

Viele Grüße aus dem Taunus!
Alex

**Comment by bilersan** [comment]
Just wanted to use this issue to also express my gratitude for this wonderful application.

Thank you kind sir :^)

**Comment by albar965** [comment]
> Just wanted to use this issue to also express my gratitude for this wonderful application.
> 
> Thank you kind sir :^)

You're very welcome @bilersan :slightly_smiling_face:
